aptoide-views/src/main/res/layout/appview_poa_countdown_element.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    >
+
+  <TextView
+      android:id="@+id/hours"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:background="@drawable/appview_countdown_rounded_corners_black_background"
+      android:textColor="@color/white"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      tools:text="24"
+      style="@style/Aptoide.TextView.Medium.L"
+      />
+
+  <TextView
+      android:id="@+id/hours_tag"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="@string/poa_time_left_hours"
+      android:textAllCaps="true"
+      android:textColor="#949494"
+      android:textSize="8sp"
```

Comment:
textsize and textcolor hardcoded here. Shouldn't we create a textstyle? (Don't know because we rarely use these sizes... but maybe it should have been extracted (?))